converter/solid_figures.py

```diff
@@ -0,0 +1,75 @@
+from dataclasses import dataclass
+from abc import ABC
+
+
+@dataclass(frozen=True)
+class SolidFigure(ABC):
+    """Abstract figure, bodies are used to define geometries."""
+
+    uuid: str
+    offset: list[float]
```

Comment:
Allowing list of any length is not a good idea, its too abstract. We should somehow enforce 3 components.
I'd suggest a dataclass for 3-element vector, with X,Y,Z component, we could name it XYZVector, as it could also be reused for rotation angles.

Reply:
If the only thing we want is for them to have fixed size wouldn't tuples suffice?

Reply:
Sure, good idea !